documentation/source/user_section/tutorials/spinal-nerve-rootlets-segmentation/before-starting.rst

```diff
@@ -0,0 +1,14 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2/t2.nii.gz``: A T2w anatomical image.
+
+   You can get these files by downloading `data_rootlets-segmentation.zip <https://github.com/spinalcordtoolbox/sct_tutorial_data/archive/refs/heads/jv/add-rootlets-files.zip>`_.
```

Comment:
For now, this PR temporarily links to the `sct_tutorial_data` repo, as found in PR https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/22.

When the SCT PR is approved, we can:

- [ ] Merge the `sct_tutorial_data` PR
- [ ] Create a new release for `sct_tutorial_data`
- [ ] Update the version number listed in `conf.py`
- [ ] Update this link as follows:

```suggestion
   You can get these files by downloading :sct_tutorial_data:`data_rootlets-segmentation.zip`.
```